scripts/configure.py

```diff
@@ -459,11 +459,21 @@ def print_progress(tasks):
     progress(tasks)
     return tasks
   tasks = []
+
+  # Comment out deb http://raspbian.raspberrypi.org/raspbian/ buster main contrib non-free rpi from /etc/apt/sources.list to avoid hitting up a now empty apt source
+  tasks += print_progress([Task('Deactivate apt updates for outdated OS',
+                                args='file=/etc/apt/sources.list; '
+                                'if grep -q "buster" "$file" && [ "$(head -c 1 "$file")" != "#" ]; then '  # If file is for rasbian buster and the first line isn't already commented out, comment out the first line
+                                ' sudo sed -i "1s/^/#/" "$file"; '
```

Comment:
I would rather be more explicit here and replace the known incorrect string that may not be in the exact place we expect.

```suggestion
                                '   sudo sed -i "s@^deb http://raspbian.raspberrypi.org/raspbian/ buster@#deb http://raspbian.raspberrypi.org/raspbian/ buster@g" "$file"; '
```

Reply:
I tested this command on lamplipi